docs/manuals/source/TasksAndConcepts/CatalogMaintenance.rst

```diff
@@ -644,9 +661,9 @@ Assuming that you start all your nightly backup jobs at 1:05 am (and that they r
      Messages = Standard
      Pool = Default
      # This creates an ASCII copy of the catalog
-     # Arguments to make_catalog_backup.pl are:
-     #  make_catalog_backup.pl <catalog-name>
-     RunBeforeJob = "/usr/lib/bareos/scripts/make_catalog_backup.pl MyCatalog"
+     # Arguments to make_catalog_backup are:
```

Comment:
It would be nice, if this config example would be closely related to our default configuration. Lines like `Storage = DLTDrive` might cause confusion.